packages/AdaptiveCards/AdaptiveCardsBotComponent.cs

```diff
@@ -13,7 +13,7 @@ namespace Microsoft.Bot.Components.AdaptiveCards
     /// <summary>
     /// <see cref="BotComponent"/> implementation for the adaptve card types.
     /// </summary>
-    public class AdaptiveBotComponent : BotComponent
+    public class AdaptiveCardsBotComponent : BotComponent
```

Comment:
I might need to revert this change depending on timing for this component. If it shipped or ships before this PR gets in, then I'll just revert the rename. Too bad that the file name is correct but the class name does not match it :(